roles/swift-account/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
     - { name: account-auditor }
     - { name: account-reaper }
     - { name: account-replicator }
+  when: ansible_os_family == 'Debian'
```

Comment:
can we stay consistent with other openstack roles (glance, nova, neutorn, heat, aodh, ceilometer, etc).
We use os == 'ubuntu' and os == 'rhel' to differentiate between family debian and family RHEL.